resources/twitchcommands.py

```diff
@@ -676,53 +689,54 @@ def gamecontrol():
             "(logclear! (-> *pc-settings* cheats) (pc-cheats no-tex))")
             message = ""
 
-        if PREFIX + "x" == str(args[0]).lower():
-          sendForm("(logior! (cpad-pressed 0) (pad-buttons x))")
-          message = ""
-        if PREFIX + "square" == str(args[0]).lower():
-          sendForm("(logior! (cpad-pressed 0) (pad-buttons square))")
-          message = ""
-        if PREFIX + "triangle" == str(args[0]).lower():
-          sendForm("(logior! (cpad-pressed 0) (pad-buttons triangle))")
-          message = ""
-        if PREFIX + "circle" == str(args[0]).lower():
-          sendForm("(logior! (cpad-pressed 0) (pad-buttons circle))")
-          message = "" 
+        if PREFIX + "press" == str(args[0]).lower() and len(args) >= 2 and on_check("press") and cd_check("press"):
+            sendForm("(logior! (cpad-pressed 0) (pad-buttons " + str(args[1]) + "))")
+            message = ""
```

Comment:
similar here, if they give a bad button name hopefully it just does nothing?

Reply:
could also be cool to support mapping a single character to the button for shorter commands (with the `press` prefix still)

[c]ircle
[d]own
[l]eft
[r]ight
[s]quare
[t]riangle
[u]p
[x]